### ISSUE TITLE
raven.middleware.express.requestHandler is not a function

### ISSUE THREAD

**Issue by dhritzkiv** [getsentry/raven-node#119]
Following the directions in the README:

````js
//near the top of the middleware stack
app.use(raven.middleware.express.requestHandler('{{ SENTRY_DSN }}'))
//near the bottom, before any other errorHandlers
app.use(raven.middleware.express.errorHandler('{{ SENTRY_DSN }}'))
````

I get `raven.middleware.express.requestHandler is not a function`, I suspect the README is out of date, as I see other documentation suggesting to simply use `raven.middleware.express('{{ SENTRY_DSN }}')

**Comment by kirushanth-sakthivetpillai** [comment]
Current master on github supports requestHandler/errorHandler.
I'm assuming you're using NPM version which does not includes these latest changes.
If you want these changes in your project, update your package.json to reflect this:
git://github.com:getsentry/raven-node.git#master

**Comment by kirushanth-sakthivetpillai** [comment]
Please close.

**Comment by dhritzkiv** [comment]
I've since learned of the discrepancy between this repo and npm – I thought npm was in lockstep, but was mistaken.

I vote to keep this issue open until npm and this repo('s README) are in sync.

**Comment by mattrobenolt** [comment]
I've published a 0.8.0 to npm which reflects this. See: https://github.com/getsentry/raven-node/blob/master/History.md#080---06152015

Sorry for the confusion. :disappointed:
